src/main/resources/db/migration/oracle/V1.6__del_old_kall_logg_partitions.sql

```diff
@@ -0,0 +1,23 @@
+CREATE OR REPLACE PROCEDURE del_old_kall_logg_partitions IS
+    CURSOR kall_logg_partitions IS
+        SELECT partition_name, high_value
+        FROM USER_TAB_PARTITIONS
+        WHERE TABLE_NAME = 'KALL_LOGG';
+BEGIN
+    FOR rec IN kall_logg_partitions
+        LOOP
+            IF to_date(substr(rec.high_value, 12, 10), 'YYYY-MM-DD') < CURRENT_DATE - 30 THEN
+                EXECUTE IMMEDIATE 'ALTER TABLE kall_logg DROP PARTITION ' || rec.partition_name || ' UPDATE INDEXES';
```

Comment:
"Specify UPDATE INDEXES in the ALTER TABLE statement. Doing so leverages the new asynchronous global index maintenance. Indexes remain valid."
Se https://docs.oracle.com/database/121/VLDBG/GUID-40D491DB-7965-4935-AB7A-F8D2DADA33C0.htm